x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/timeline/infinity_icon.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
+  <defs/>
+  <path fill="#000" fill-rule="evenodd" d="M5.44354 9.86928c-.47429.46802-1.12585.75702-1.84475.75702-1.45038 0-2.626143-1.17576-2.626143-2.62614S2.14841 5.37401 3.59879 5.37401c.71899 0 1.37049.28893 1.84473.75702l.00033-.00033.27438.27438L7.31327 8.0001l-.00001.00001.68777.68777.00001-.00002 1.86843 1.86844-.00078.0007c.65021.6442 1.54481 1.042 2.53251 1.042C14.3888 11.599 16 9.98772 16 8.00016s-1.6112-3.59879-3.5988-3.59879c-.9876 0-1.8822.39779-2.53249 1.04188l.00072.00073-.76518.76516.68777.68777.79978-.79975-.0002-.00017c.471-.44801 1.1082-.72298 1.8096-.72298 1.4504 0 2.6262 1.17577 2.6262 2.62615s-1.1758 2.62614-2.6262 2.62614c-.7012 0-1.3384-.2749-1.8094-.72282l.0002-.00024L8.68881 8.0001l.00278-.00277-.68777-.68777-.00278.00278-1.90297-1.90295-.0007.0007c-.64728-.62456-1.52806-1.00872-2.49858-1.00872C1.61124 4.40137 0 6.0126 0 8.00016 0 9.98772 1.61124 11.599 3.59879 11.599c.97055 0 1.85122-.3841 2.49844-1.0086l.00065.0006.80272-.8027-.68777-.68777-.76902.76901-.00027-.00026z" clip-rule="evenodd"/>
```

Comment:
You'll need to remove this fill and let the EuiIcon handle the fill through CSS or else it won't be visible in dark mode.
```suggestion
  <path fill-rule="evenodd" d="M5.44354 9.86928c-.47429.46802-1.12585.75702-1.84475.75702-1.45038 0-2.626143-1.17576-2.626143-2.62614S2.14841 5.37401 3.59879 5.37401c.71899 0 1.37049.28893 1.84473.75702l.00033-.00033.27438.27438L7.31327 8.0001l-.00001.00001.68777.68777.00001-.00002 1.86843 1.86844-.00078.0007c.65021.6442 1.54481 1.042 2.53251 1.042C14.3888 11.599 16 9.98772 16 8.00016s-1.6112-3.59879-3.5988-3.59879c-.9876 0-1.8822.39779-2.53249 1.04188l.00072.00073-.76518.76516.68777.68777.79978-.79975-.0002-.00017c.471-.44801 1.1082-.72298 1.8096-.72298 1.4504 0 2.6262 1.17577 2.6262 2.62615s-1.1758 2.62614-2.6262 2.62614c-.7012 0-1.3384-.2749-1.8094-.72282l.0002-.00024L8.68881 8.0001l.00278-.00277-.68777-.68777-.00278.00278-1.90297-1.90295-.0007.0007c-.64728-.62456-1.52806-1.00872-2.49858-1.00872C1.61124 4.40137 0 6.0126 0 8.00016 0 9.98772 1.61124 11.599 3.59879 11.599c.97055 0 1.85122-.3841 2.49844-1.0086l.00065.0006.80272-.8027-.68777-.68777-.76902.76901-.00027-.00026z" clip-rule="evenodd"/>
```